src/types.rs

```diff
@@ -651,34 +693,78 @@ impl Term {
                 .collect(),
         }
     }
-    /// The head of the Term.
+    /// Returns the head of the `Term`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, Term, parse_term, Atom};
+    ///
+    /// let mut sig = Signature::default();
+    ///
+    /// let op = sig.new_op(2, Some("S".to_string()));
+    /// let t = parse_term(&mut sig, "S(y_ z_)").expect("parse of S(y_ z_)");
+    ///
+    /// assert_eq!(t.atoms().len(), 3);
+    /// assert_eq!(t.head(), Atom::Operator(op));
+    /// ```
     pub fn head(&self) -> Atom {
         match self {
             Term::Variable(v) => Atom::Variable(*v),
             Term::Application { op, .. } => Atom::Operator(*op),
         }
     }
-    /// The head of the Term.
+    /// Returns the arguments of the `Term`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, Term, parse_term, Atom};
+    /// let mut sig = Signature::default();
+    ///
+    /// let t = parse_term(&mut sig, "S K").expect("parse of S K");
+    /// let arg0 = parse_term(&mut sig, "S").expect("parse of S");
+    /// let arg1 = parse_term(&mut sig, "K").expect("parse of K");
+    /// assert_eq!(t.atoms().len(), 3);
+    /// assert_eq!(t.args(), vec![arg0, arg1]);
+    ///
+    /// let t2 = parse_term(&mut sig, "S").expect("parse of S");
+    /// assert_eq!(t2.atoms().len(), 1);
+    /// assert_eq!(t2.args(), vec![]);
+    ///
+    /// let t3 = parse_term(&mut sig, "S(y_)").expect("parse of S(y_)");
+    /// let arg = Term::Variable(t3.variables()[0]);
+    /// assert_eq!(t3.atoms().len(), 2);
+    /// assert_eq!(t3.args(), vec![arg]);
+    /// ```
     pub fn args(&self) -> Vec<Term> {
         match self {
             Term::Variable(_) => vec![],
             Term::Application { args, .. } => args.clone(),
         }
     }
-    /// Every subterm and its place, starting with the original term itself.
+    /// Returns every subterm and its place, starting with the original `Term` itself.
```

Comment:
How about ``Every subterm and its [`Place`], starting with `self` and the empty [`Place`].``?